mvvmfx/src/main/java/de/saxsys/mvvmfx/Initialize.java

```diff
@@ -0,0 +1,16 @@
+package de.saxsys.mvvmfx;
```

Comment:
Please add the licence header with your name above the package declaration.

/*******************************************************************************
 * Copyright 2017 <your name>
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 ******************************************************************************/